docs/rst/knora-api-server/api_v1/adding-resources.rst

```diff
@@ -132,70 +126,232 @@ The resource's Iri has to be provided in the URL (as its last segment). The new
+            </biblio:publicationHasAuthor>
+            <biblio:publicationHasDate knoraType="date_value">GREGORIAN:1976</biblio:publicationHasDate>
+            <biblio:publicationHasTitle knoraType="richtext_value">Strings in the 16th and 17th Centuries</biblio:publicationHasTitle>
+            <biblio:publicationHasTitle knoraType="richtext_value">An alternate title</biblio:publicationHasTitle>
+            <biblio:startPage knoraType="richtext_value">48</biblio:startPage>
+        </biblio:JournalArticle>
+    </knoraXmlImport:resources>
+
+This illustrates several aspects of XML imports:
+
+- The root XML element must be ``knoraXmlImport:resources``.
+- There is an XML namespace corresponding each ontology used in the import. These namespaces can be found in the
+  XML schema files returned by the Knora API server.
+- We have copied and pasted ``xmlns="http://api.knora.org/ontology/biblio/xml-import/v1#"`` from the main XML schema,
+  ``biblio.xsd``. This enables the Knora API server to identify the main ontology we are using.
+- We have used ``xsi:schemaLocation`` to indicate the main schema's namespace and filename. If we put our XML document in
+  the same directory as the schemas, and we run an XML validator to check the XML, it should load the schemas.
+- The child elements of ``knoraXmlImport:resources`` represent resources to be created. The order of these elements
+  is unimportant.
+- Each resource must have an ID, which must be an XML NCName_, and must be unique within the file. These IDs are used only during the import,
+  and will not be stored in the triplestore.
+- The first child element of each resource must be a ``knoraXmlImport:label``, which will be stored as the resource's ``rdfs:label``.
+- Optionally, the second child element of a resource can provide metadata about a file to be attached to the resource
+  (see :ref:`bulk-import-with-digital-representations`).
+- The remaining child elements of each resource represent its property values. These must be sorted in alphabetical order by
+  property name.
+- If a property has mutliple values, these are represented as multiple adjacent property elements.
+- The type of each value must be specified using the attribute ``knoraType``.
+- A link to another resource described in the XML import is represented as a child element of a property element,
+  with attributes ``knoraType="link_value"`` and ``linkType="ref"``, and a ``target`` attribute containing
+  the ID of the target resource.
+- There is a specfic syntax for referring to properties from other ontologies. In the example, ``beol:comment``
+  is defined in the ontology ``http://www.knora.org/ontology/beol``. In the XML, we refer to it as
+  ``biblio:beol__comment``.
+- A text value can contain XML markup. In this case:
+    - The text value element must have the attribute ``mapping_id``, specifying a mapping from XML to standoff markup (see :ref:`XML-to-standoff-mapping`).
```

Comment:
The mapping_id is only needed in case some markup is provided, right? If there is just plain text, no mapping needs to be indicated.